src/test/java/org/jenkinsci/plugins/ownership/model/workflow/OwnershipGlobalVariableTest.java

```diff
@@ -0,0 +1,166 @@
+        OwnershipDescription d = OwnershipDescription.DISABLED_DESCR;
+        WorkflowRun run = buildSnippetAndAssertSuccess("printJobOwnershipInfo", d, false);
+        assertThat(run.getLog(), containsString("Ownership is disabled"));
+    }
+    
+    @Test
+    public void jobOwnershipSnippet_noOwnership() throws Exception {
+        OwnershipDescription d = OwnershipDescription.DISABLED_DESCR;
+        WorkflowRun run = buildSnippetAndAssertSuccess("printJobOwnershipInfo", d, true);
+        assertThat(run.getLog(), containsString("Ownership is disabled"));
+    }
+    
+    @Test
+    public void shouldProvideJobOwnershipInfo_sandbox() throws Exception {
+        OwnershipDescription d = new OwnershipDescription(true, "owner",
+                Arrays.asList("coowner1", "coowner2"));
+        WorkflowRun run = buildSnippetAndAssertSuccess("printJobOwnershipInfo", d, false);
+        assertThat(run.getLog(), containsString("owner"));
+    }
+    
+    @Test
+    public void shouldProvideJobOwnershipInfo() throws Exception {
+        OwnershipDescription d = new OwnershipDescription(true, "owner",
+                Arrays.asList("coowner1", "coowner2"));
+        WorkflowRun run = buildSnippetAndAssertSuccess("printJobOwnershipInfo", d, true);
+        assertThat(run.getLog(), containsString("owner"));
+    }
+    
+    @Test
+    public void nodeOwnershipSnippet_onSlave() throws Exception {
+        j.jenkins.setLabelString("requiredLabel");
+        DumbSlave slave = j.createOnlineSlave(new LabelAtom("requiredLabel"));
+        NodeOwnerHelper.setOwnership(slave, new OwnershipDescription(true, "ownerOfJenkins",
+                Arrays.asList("coowner1", "coowner2")));
+        OwnershipDescription d = OwnershipDescription.DISABLED_DESCR;
+        WorkflowRun run = buildSnippetAndAssertSuccess("printNodeOwnershipInfo", d, false);
+        assertThat(run.getLog(), containsString("ownerOfJenkins"));
+    }
+    
+    @Test
+    public void nodeOwnershipSnippet_onMaster() throws Exception {
+        j.jenkins.setLabelString("requiredLabel");
+        NodeOwnerHelper.setOwnership(j.jenkins, new OwnershipDescription(true, "ownerOfJenkins",
+                Arrays.asList("coowner1", "coowner2")));
+        OwnershipDescription d = OwnershipDescription.DISABLED_DESCR;
+        WorkflowRun run = buildSnippetAndAssertSuccess("printNodeOwnershipInfo", d, true);
+        assertThat(run.getLog(), containsString("ownerOfJenkins"));
+    }
+    
+    @Test
+    public void nodeOwnershipSnippet_onMaster_sandbox() throws Exception {
+        j.jenkins.setLabelString("requiredLabel");
+        NodeOwnerHelper.setOwnership(j.jenkins, new OwnershipDescription(true, "ownerOfJenkins",
+                Arrays.asList("coowner1", "coowner2")));
+        OwnershipDescription d = OwnershipDescription.DISABLED_DESCR;
+        WorkflowRun run = buildSnippetAndAssertSuccess("printNodeOwnershipInfo", d, false);
```

Comment:
Huh? You are passing `false` for `sandbox` yet naming the test `*_sandbox`?

Generally there is little point in testing with `sandbox=false`. Always test with it true.